package.json

```diff
@@ -2,20 +2,22 @@
   "name": "jest-clean-console-reporter",
   "version": "0.2.0",
   "description": "Jest reporter for grouping and filtering intrusive console warnings",
-  "main": "lib/index.js",
+  "main": "lib/index.ts",
```

Comment:
I'm not 100% sure about this, but I think the package main needs to point to the built .js file in dist/.

Reply:
Correct

Reply:
Yup it does! I hadn't changed this yet because I was waiting to see what we were going to do about using tsdx or nx or what. All I did last night was switch to TS